README.md

```diff
@@ -1 +1,6 @@
-`deno run -A --watch --unstable server.js `
\ No newline at end of file
+`deno run -A --watch --unstable server.js `
+
+To run with import maps
+```shell
+deno run --allow-net --allow-read --import-map=importmap.json --watch --unstable  server.js
```

Comment:
Ahh, that's a bummer then for the `hydration` part. I will try and see the `deps` approach. But, not sure. Here is the reason, like if we use `url` imports, we will be loading modules using something like this.

```js
import { jsx } from 'http://ga.jspm.io/nano-jsx/index.js
```

And for hydration we load the same module. In this case, any sub-deps that `nano-jsx/index.js` is using are not resolved. Since, the resolution relies on `import-maps` again 😅